request-fetch-adapter.js

```diff
@@ -28,4 +28,18 @@ module.exports = {
       callback(err);
     }
   }
-};
\ No newline at end of file
+};
+
+function getHttpMethodName(verb) {
```

Comment:
Should add a comment indicating that this should stay in sync with the CodePush SDK.  It's not obvious here that the array below should be synchronized with the SDK's Verb enum (https://github.com/Microsoft/code-push/blob/master/sdk/script/acquisition-sdk.ts#L6).